app/package.json

```diff
@@ -163,13 +149,11 @@
     "@types/dotenv": "^8.2.0",
     "@types/exceljs": "^1.3.0",
     "@types/file-saver": "^2.0.1",
-    "@types/fuzzaldrin-plus": "^0.6.1",
     "@types/github-slugger": "^1.3.0",
     "@types/graphql-type-json": "^0.3.2",
     "@types/jest": "^27.0.2",
     "@types/k6": "^0.45.0",
     "@types/lodash": "^4.14.165",
-    "@types/micro-cors": "^0.1.1",
```

Comment:
Thanks for removing all the unnecessary deps. How did you detect them ? With the sheer power of reading + searching 😁 ?